projects/fishtornado/Ocean.h

```diff
@@ -22,14 +22,15 @@ using namespace ppx;
 #include "Buffer.h"
 
 class FishTornadoApp;
+struct FishTornadoSettings;
 
 class Ocean
 {
 public:
     Ocean();
     ~Ocean();
 
-    void Setup(uint32_t numFramesInFlight);
+    void Setup(uint32_t numFramesInFlight, const FishTornadoSettings& settings);
```

Comment:
Why do ocean and shark need settings?

Reply:
They don't yet. Can eliminate this part; I just liked the symmetry.

Reply:
Do you anticipate they would use it in the future? I'd probably keep them if that's the case and remove them otherwise, but up to you

Reply:
No value in code that anticipates for a future that doesn't exist. I'll clean it up, I think.